docs/UserGuide.md

```diff
@@ -140,11 +140,54 @@ Examples:
 * `delmod m/CS2103` deletes the existing module with code `CS2103` from FaculType.
 * `delmod m/CS2102` deletes the existing module with code `CS2102` from FaculType.
 
-### Clearing all entries : `clear`
+### Finding modules : `findmod`
 
-Clears all entries from the FaculType.
+Finds all  modules whose code or name contain the given keyword.
```

Comment:
"keywords" might be better since we can have multiple keywords 😁

Reply:
Make sense. I'll change that.